src/components/business/business.jsx

```diff
@@ -1,20 +1,33 @@
 import React from "react";
 import Queue from "./queue.jsx";
-import PreviousOrder from "./prevorder.jsx";
+import PreviousOrders from "./prevorders.jsx";
 
 class Business extends React.Component {
   state = {
-    active: {},
-    queue: [],
-    complete: []
+    view: 'queue'
   };
 
+  changeView(option) {
+    this.setState({
+      view: option
+    });
+  }
+
+  renderView() {
+    const { view } = this.state;
+    if (view === 'queue') {
```

Comment:
@jpdaprato I see you used the routing from the assessment! Nice! You may want to give Reach Router a shot as it would solve these things for you without needing to change the view like you set up. Just a suggestion as it's more "real world"